### ISSUE TITLE
Allowing configuration of the displayed glyphs per controller and glyph fallback.

### ISSUE THREAD

**Issue by Daruyami** [isXander/Controlify#341]
**Is your feature request related to a problem? Please describe.**

When using generic controllers, or controller scheme emulation, for example in a situation like [this](https://github.com/isXander/Controlify/issues/336), the shown glyphs may not match the actual controllers glyphs, or just the users' preferred glyphs.

Furthermore, in some cases (Dualsense Edge emulation) the detected glyphs may not encompass the entirety of a controllers input scheme, causing some glyphs to be missing:
![image](https://github.com/isXander/Controlify/assets/67425425/847337dc-81de-4b3f-bd1d-07189b8b92e4)
^image depicts an assigned right paddle input for an action with PS glyphs which shows as a _rectangle_.

**Describe the solution you'd like**

Option in the settings to switch between available glyph sets (or maybe choosing glyphs for each and every input?) and fallback to other glyph sets for a button input with a missing glyph.

**Describe alternatives you've considered**

@isXander suggested on the Discord server making a resource pack as a workaround, which is better than nothing, however not very user friendly.

Another alternative could be showing both types of glyphs for an action like done in some games (I can't remember of an example, sorry :frowning_face:).

**Comment by RobinYH** [comment]
I agree with this, I would like to see the ability to select which glyphs are shown. I use an 8BitDo controller through Steam Input, and the icons used for it don't really work that well (for example, the L2 and R2 buttons are just labeled as L and R with a different button shape). It would be nice to be able to pick between Xbox, Playstation, Steam and Switch icons.
